### ISSUE TITLE
Best usage for different modes?

### ISSUE THREAD

**Issue by HinoReo** [AHAAAAAAA/PokemonGo-Map#2018]
I see two ways of using pogomaps:

1. Individual (setting up for oneself only), where current user location and changing it on the fly matters a lot, along with changing steps and all this, but probably without multiple instances and/or threads. In other words, it should be as simple to set up as possible.

2. Community (setting up to use with friends), where current user location should not affect the scanning process. Mostly, it means MySQL + multiple instances + as little GET requests as we can get + none of POST (we don't want neither to know every user location nor give the ability to change script center point and/or amount of steps to scan).

TL;DR How do I disable all POST requests, but allow displaying of "blue dot"?

**Comment by hauxir** [comment]
I basically have the post requests blocked in my nginx config but allowing the blue dot would be nice.

**Comment by coolaj86** [comment]
I support multiple users on my node.js fork, which uses the same UI:

* <https://github.com/Daplie/node-pokemap>

My account got soft-blocked while I was adding the pokestops so I haven't been able to finish testing, but I added all of the data so it should work. :)

Also, the scanning is lot as far reaching yet, but that can be accomplished by dragging the location marker.

**Comment by HinoReo** [comment]
@coolaj86 
> dragging the location marker
Sorry, that is exactly what I would like to avoid

**Comment by Pedrock** [comment]
@HinoReo Use --fixed-location
